src/__tests__/ReactHardwareComponent-test.js

```diff
@@ -25,55 +30,45 @@ describe('ReactHardwareComponent', () => {
     });
   });
 
-  describe('receiveComponent', function() {
-    let gen;
-    beforeEach(function() {
-      const NodeStub = function(initialProps) {
-        this._currentElement = {props: initialProps};
-        this._rootNodeID = 'test';
-      };
-      NodeStub.prototype = new ReactHardwareComponent({
-        validAttributes: {
-          pin: true,
-          value: true,
-        },
+  describe('pin', () => {
```

Comment:
I don't think so. I just added it that way because the tests only tested the pin component. 
I'll revert it back. :+1: